### ISSUE TITLE
Crossing 0 longitude

### ISSUE THREAD

**Issue by ryanreisinger** [ianjonsen/foieGras#3]
When individuals cross 0 degree longitude, the estimated locations are at -180/+180 longitude.

Fix for #2 might have caused a new issue.

```
[lmsa.zip](https://github.com/ianjonsen/foieGras/files/2850289/lmsa.zip)

library(foieGras)
library(dplyr)

dat <- read.csv("lmsa.csv") #in zip file attached.

output <- foieGras::fit_ssm(d = select(dat, id, date, lc, lon, lat),
                            vmax = 36,
                            min.dt = 120,
                            time.step = 2)

plot(output$ssm[[1]], est = "predicted")
```

**Comment by ryanreisinger** [comment]
![rplot](https://user-images.githubusercontent.com/13516646/52552939-48dfdc00-2de2-11e9-875a-553714deec7b.png)

**Comment by ianjonsen** [comment]
I've uploaded a fix that works for this track and the previous example track crossing -180/180. I've also added a lonWrap argument to the plot method where user can specify a minimum longitude for wrapping: 0 to deal with dateline issue, or -180 to deal with crossing 0,360. The default is no wrapping.

I'll leave this issue open for now, in case you run into more problems

![lmsa_ex](https://user-images.githubusercontent.com/9984184/52813508-4db9bf80-3070-11e9-8787-78c78eb39545.png)
